ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-commands.sh

```diff
@@ -1,37 +1,50 @@
 #!/bin/bash
 
-# Generates a workspace name like rdr-multi-arch-4-14-au-syd-n1
-# this keeps the workspace unique per version
-OCP_VERSION=$(cat "${SHARED_DIR}/OCP_VERSION")
-REGION="${LEASED_RESOURCE}"
-CLEAN_VERSION=$(echo "${OCP_VERSION}" | tr '.' '-')
-if [ ! -z "${WORKFLOW_TYPE}" ]
+##### Constants
+IBMCLOUD_HOME=/tmp/ibmcloud
+export IBMCLOUD_HOME
```

Comment:
How is this variable used?  I don't see "`IBMCLOUD_HOME`" used in this script at all, but is it used from the environment variables for commands executed or something?

Reply:
it's pickedup by the CLI

Reply:
Answered async in slack.  Thanks!